### PR TITLE
Add unit tests to Apache Kafka hooks

### PR DESCRIPTION
**Pull request by shahar1**
related: #35127

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/providers/apache/kafka/hooks/test_consume.py [hunk 1]
```diff
@@ -52,13 +52,8 @@ def setup_method(self):
...
-        """test initialization of AdminClientHook"""
+    def test_get_consumer(self):
+        assert self.hook.get_consumer() == self.hook.get_conn
 
-        # Standard Init
-        KafkaConsumerHook(["test_1"], kafka_config_id="kafka_d")
-
-        # Not Enough Args
-        with pytest.raises(ValueError):
-            KafkaConsumerHook(["test_1"], kafka_config_id="kafka_bad")
```
Comment (shahar1) [inline tests/providers/apache/kafka/hooks/test_consume.py]: This one fails, so I had to remove it

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Conflicts.

**Comment by shahar1** [comment]
> Conflicts.

Apologies for the confusion - this PR should have been merged before #40142, as the latter already includes this one's commit.
Basically, there are now 0 files to merge 😅
I guess that it should either be closed or merged for the protocol.

**Comment by eladkal** [comment]
Then lets just close

**Pull request closed without merging**